sql/parser/parser_test.go

```diff
@@ -43,3 +44,9 @@ func TestParserMultiStatement(t *testing.T) {
 		})
 	}
 }
+
+func TestParserDivideByZero(t *testing.T) {
+	require.NotPanics(t, func() {
+		_, _ = ParseQuery(context.Background(), "SELECT*FROM t LIMIT-0%.2")
```

Comment:
A bit more human-readable test case would be
```suggestion
		_, _ = ParseQuery(context.Background(), "SELECT * FROM t LIMIT 0 % .5")
```